Formula/gcalcli.rb

```diff
@@ -0,0 +1,141 @@
+class Gcalcli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Easily access your Google Calendar(s) from a command-line"
+  homepage "https://github.com/insanum/gcalcli"
+  url "https://github.com/insanum/gcalcli/archive/v4.3.0.tar.gz"
+  sha256 "597d96ee1c8b5f094cca75a059cb1a0056a612e734c32012ef0d41219ac3b0cd"
+  license "MIT"
+  head "https://github.com/insanum/gcalcli.git"
+
+  livecheck do
+    url :stable
+  end
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "ea18ab2445955c20de54537447b5930de60cf8e927fe291f662760d50c0df8db" => :catalina
+    sha256 "737880ee8c7e9d73848eec50d87b1b7da7762a8b8d6172fd1f4be173888e18fe" => :mojave
+    sha256 "4d2c498462eba573c43413ec16f4681a8e604b67ff6739351900e500e2be5923" => :high_sierra
+  end
+
+  depends_on "python@3.8"
```

Comment:
Does it not work with the latest python?

Reply:
It does - does it mean I can remove this? I tried using "python@3.x" as described in https://docs.brew.sh/Python-for-Formula-Authors but it failed with `No available formula with the name "python@3.x"`

Reply:
No, it means this should be 
```suggestion
  depends_on "python@3.9"
```

Reply:
Ah ok! Sure - will update

Reply:
updated to latest version